modules/light-clients/08-wasm/types/contract_api.go

```diff
@@ -96,55 +96,35 @@ type verifyUpgradeAndUpdateStateMsg struct {
 type checkSubstituteAndUpdateStateMsg struct{}
 
 // ContractResult defines the expected interface a Result returned by a contract call is expected to implement.
-type ContractResult interface {
-	Validate() bool
-	Error() string
-}
+type ContractResult interface{}
```

Comment:
I don't think this interface provides any value now? we can just use built-in `any` or `interface{}`

Reply:
It might make sense to just use it as a type alias now? Would read better in the generic `call/query` funcs